README.md

```diff
@@ -11,8 +11,11 @@ A [hapi](https://github.com/hapijs/hapi) plugin integrating [Objection ORM](http
 Schwifty is used to define [Joi](https://github.com/hapijs/joi)-compatible models and knex connections for use with Objection ORM.  Those models then become available within your hapi server where it is most convenient.  It has been tailored to multi-plugin deployments, where each plugin may set clear boundaries in defining its own models, knex database connections, and migrations.  It's safe to register schwifty multiple times, wherever you'd like to use it, as it protects against model name collisions and other ambiguous configurations.
 
 ```js
-// First, ensure your project includes your
-// preferred versions of knex, objection, and sqlite3
+// First, ensure your project includes knex, objection, and sqlite3
+// Note that for knex and Objection, we assume using
+// knex >=0.8 and Objection >=1 <2 (see our peerDependencies in our package file)
+// Schwifty v4 introduced compatibility with Objection v.1
```

Comment:
It might just be me, but `v.1` reads like `v0.1` to me, not `v1.0` -- might be good to tweak this to disambiguate. Maybe just `v1`?

Reply:
Agree 👍

Reply:
👌 mm! dah, good point. thanks, y'all, fixing now 🙏